.github/workflows/build.yml

```diff
@@ -17,7 +17,8 @@ on:
       - "**/*.go"
       - Dockerfile
       - .github/workflows/build.yml
-
+    tags:
```

Comment:
This won't work since the tag is created by release-please (using the default `github.token`) and so follow-up automations will not be triggered. You will need to explicitly call this workflow from the release workflow 🙂